src/main/java/com/mongodb/jdbc/MongoConnection.java

```diff
@@ -48,7 +49,16 @@ public MongoConnection(ConnectionString cs, String database, String conversionMo
         this.url = cs.getConnectionString();
         this.user = cs.getUsername();
         this.currentDB = database;
-        mongoClient = MongoClients.create(cs);
+
+        StringBuilder version = new StringBuilder();
+        version.append(MongoDriver.MAJOR_VERSION).append(".").append(MongoDriver.MINOR_VERSION);
```

Comment:
I think we want the whole version (not just major and minor) here -- AFAICT, the only reason we've broken these two version components out into constants and not others is because the JDBC API only has methods for major & minor.

Reply:
Sounds good.  
I was looking for a way to use the default value that MAJOR_VERSION and MINOR_VERSION specifies if VERSION is `null` (currently set to 0.0).  I can add a check to use VERSION if it is not null and MAJOR_VERSION.MINOR_VERSION in case it is null.